check_sites.py

```diff
@@ -18,33 +18,63 @@
 import os
 from publicsuffix2 import PublicSuffixList
 
+def find_diff_sets(old_sets, new_sets):
+    """Finds changes made between two dictionaries of First-Party Sets
+
+        Finds First-Party Sets that have been added or modified in old_sets
+        to create new_sets and returns them as the dictionary diff_sets. 
+        Additionally, finds First-Party Sets that have been removed from
+        old_sets to create new_sets and returns them as subtracted_sets.
+
+        Args:
+            old_sets: a dictionary of primary->FpsSet
+            new_sets: a dictionary of primary->FpsSet
+        Returns:
+            diff_sets: a dictionary of primary->FpsSet
+            substracted_sets: a dictionary of primary->FpsSet
```

Comment:
```suggestion
            old_sets: Dict[string, FpsSet]
            new_sets: Dict[string, FpsSet]
        Returns:
            Tuple[Dict[string, FpsSet], Dict[string, FpsSet]]
```

It's probably best if we use python's standard [type hint](https://docs.python.org/3/library/typing.html) syntax.

Reply:
Implemented here, but will need to change this in other docstrings in future PR